components/edge-functions/pdforge.mdx

```diff
@@ -0,0 +1,150 @@
+---
+
+<img src="/images/banners/pdforge.png" className="rounded-lg" noZoom />
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Find it on GitHub: [<Icon icon="github" iconType="solid" /> /edgee-cloud/pdforge-generate-pdf-component](https://github.com/edgee-cloud/pdforge-generate-pdf-component)
+
+A WebAssembly component that integrates with PDForge.com to trigger PDF generation directly from the edge. This component provides a simple way to generate PDFs using PDForge's API through an HTTP endpoint.
+
+## Features
+
+- **Edge-Based PDF Generation**: Trigger PDF creation directly from the edge
+- **PDForge Integration**: Connect to PDForge.com's PDF generation service
+- **Simple HTTP Interface**: Easy-to-use endpoint for PDF generation
+- **WebAssembly Performance**: Fast, secure execution in a sandboxed environment
+- **Configurable Endpoint**: Set custom paths for your PDF generation endpoint
+
+## Getting Started
+
+**To integrate PDForge with your project:**
+
+1. Sign up for a PDForge account at [pdforge.com](https://pdforge.com)
+2. Open the Edgee console and navigate to your project's Components
+3. Select "Add a component" and choose `pdforge` from the list of available edge functions
+4. Configure the following settings:
+   - **Path Configuration**: Set the PDF generation endpoint path (e.g., `/generate-pdf`)
+   - **Component Settings**: Configure any PDForge-specific options
+5. Click Save to activate the PDF generation endpoint
+
+## Configuration Options
+
+### Required Settings
+- **Endpoint Path**: The path for PDF generation requests (e.g., `/generate-pdf`)
+- Additional PDForge-specific configuration options will be documented as the component develops
+
+### Setup Requirements
+- Active PDForge account at [pdforge.com](https://pdforge.com)
+- PDForge API credentials (will be configured through the console)
+
+<Note>
+  This component is in early development. More configuration options and documentation will be added as the component matures.
+</Note>
+
+## Usage
+
+Once configured, you can trigger PDF generation by sending requests to your configured endpoint:
+
+```javascript
+// Example usage (specific API format to be documented)
+await fetch('/generate-pdf', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json'
+  },
+  body: JSON.stringify({
+    // PDF generation parameters
+    // Specific format depends on PDForge API requirements
+  })
+});
+```
+
+<Note>
+  Detailed API documentation will be provided as the component development progresses.
+</Note>
+
+
+## Use Cases
+
+1. **Invoice Generation**: Create professional invoices
+2. **Report Generation**: Generate business reports
+3. **Certificate Creation**: Issue certificates and diplomas
+4. **Document Automation**: Automate document creation processes
+5. **Edge-Based PDF Services**: Provide PDF generation closer to users
+
+## Template Data Binding
+
+PDForge uses a simple template binding system:
```

Comment:
@NicolasGirardot can you please double-check this? It looks fine to me, but I haven't worked with their template stuff